.github/workflows/integration.yml

```diff
@@ -55,6 +55,8 @@ jobs:
           retention-days: 3
       - name: Setup LXD
         uses: canonical/setup-lxd@v0.1.2
+        with:
+          channel: latest/edge
```

Comment:
Let's make this `6/edge` instead of latest edge. It already contains the latest commit of `latest/edge` channel anyway. And limiting to `6/edge` ensures we don't see a known breaking change  across major semver to 7/edge.